### PR TITLE
Compatability update for 0.209.9

### PR DESCRIPTION
**Pull request by Nickwwest**
Reflection to access newly protected members, perf hit is negligible.

### CODE REVIEW

**Comment by Nickwwest** [comment]
I did minimal testing, spawned a flamberge, tested animations and the alternate attack and it all worked fine.  
Thanks for the great mod!

**Comment by Hugo-the-Dwarf** [comment]
Have you tried using Publized .dlls from Valheim, and the stripped corlibs from bepinex? I'm asking as some of those protected/private fields can be accessed just fine in code, and when they get patched.

Altho looking at the patch for my weapon effect fixer, I should be using sharedMaterial and not material.

I'll have to crack open my solution, re-pub the game's .dlls and update bepinex if it needs to (I'm assuming yes since they updated their unity version) and I'll see if I'm just spouting nonsense or not.

File: Armory/ValheimMoreTwoHanders/RecipePatch.cs [hunk 1]
```diff
@@ -10,7 +11,19 @@ class RecipePatch
...
+            }
+
+            try
+            {
+                __result = MyReferences.prefabsThatUpgradeAtLevelOneAlways.Contains(___m_item.name.GetStableHashCode()) ? 1 : __result;
+            }
+            catch (Exception)
+            {
+                // ignored
+            }
```
Comment (Nickwwest) [inline Armory/ValheimMoreTwoHanders/RecipePatch.cs:26]: This section is where the bug was.  When accessing the crafting table, there would be a NPE from `__result = MyReferences.prefabsThatUpgradeAtLevelOneAlways.Contains(___m_item.name.GetStableHashCode()) ? 1 : __result;` and I have no idea why.  I tried some basic defense at the start of the method then just the standard try/catch paranoia.

**Comment by Nickwwest** [comment]
No worries!  Thanks for fixing the issue, love the mod.

**Pull request closed without merging**